cumulusci/core/config/project_config.py

```diff
@@ -234,27 +236,51 @@ def _log_detected_overrides_as_warning(self, info):
     def _split_repo_url(self, url):
         url_parts = url.rstrip("/").split("/")
         name = url_parts[-1]
-        owner = url_parts[-2]
+        owner = url_parts[-2].split(":")[-1]
```

Comment:
`_apply_repo_env_var_overrides()` was the only other method that was using `_split_repo_url()`. I don't *think* that this change impacts it by changing the return value of 'owner', but you might want to double check as well.

Reply:
@Br4nd0R let's just make sure we have tests for this that cover both https and ssh git URLs

Reply:
sounds good.